BROWSER-SUPPORT.md

```diff
@@ -1,7 +1,5 @@
 # Browser support
 
-## Introduction
-
 The UI-Kit team needs to support all of our users, regardless of their device, web browser or other user agent.
 
 Alongside accessibility, extensive browser and device support is a legal requirement under the <a href="https://www.legislation.gov.au/Latest/C2016C00763" rel="external">Disability Discrimination Act (1992)</a>, requiring equal access to information on laws and government programs.
```

Comment:
I added this since we’ve referenced our *accessibility obligations* previously, whereas the browser support philosophy/policy we’re doing here — I feel —overlaps, but also extends on our accessibility obligations.

I guess I see this as a guiding standard/policy document, and ref’ing the Act that informs our requirements seemed natural to me?

Worth noting that this document should be a page/section/… somewhere within the Design Guide when we move away from KSS.